crates/dbsp/Cargo.toml

```
@@ -59,8 +59,9 @@ rand_xoshiro = "0.6.0"
 indicatif = "0.17.0-rc.11"
 clap = { version = "3.2.8", features = ["derive", "env"] }
 reqwest = { version = "0.11.11", features = ["blocking"] }
-time = { version = "0.3.14", features = [
+time = { version = "0.3.20", features = [
     "formatting",
+    "macros",
     "serde",
     "serde-human-readable",
 ] }
```

Probably should break this into a table, it'll need to be moved below the other deps first

```suggestion
[dependencies.time]
version = "0.3.20"
features = [
    "formatting",
    "macros",
    "serde",
    "serde-human-readable",
]
```